diff --git a/libselinux/src/Makefile b/libselinux/src/Makefile
index 2408fae..5efa236 100644
--- libselinux/src/Makefile
+++ libselinux/src/Makefile
@@ -183,7 +183,8 @@ install: all
 	install -m 755 $(LIBSO) $(SHLIBDIR)
 	test -d $(LIBDIR)/pkgconfig || install -m 755 -d $(LIBDIR)/pkgconfig
 	install -m 644 $(LIBPC) $(LIBDIR)/pkgconfig
-	ln -sf --relative $(SHLIBDIR)/$(LIBSO) $(LIBDIR)/$(TARGET)
+	# assumption: $(SHLIBDIR) == $(LIBDIR)
+	ln -sf $(LIBSO) $(LIBDIR)/$(TARGET)
 
 install-pywrap: pywrap
 	test -d $(PYSITEDIR)/selinux || install -m 755 -d $(PYSITEDIR)/selinux
diff --git a/libsepol/src/Makefile b/libsepol/src/Makefile
index 819d261..25844ac 100644
--- libsepol/src/Makefile
+++ libsepol/src/Makefile
@@ -86,7 +86,8 @@ install: all
 	install -m 755 $(LIBSO) $(SHLIBDIR)
 	test -d $(LIBDIR)/pkgconfig || install -m 755 -d $(LIBDIR)/pkgconfig
 	install -m 644 $(LIBPC) $(LIBDIR)/pkgconfig
-	$(LN) -sf --relative $(SHLIBDIR)/$(LIBSO) $(LIBDIR)/$(TARGET)
+	# assumption: $(SHLIBDIR) == $(LIBDIR)
+	$(LN) -sf $(LIBSO) $(LIBDIR)/$(TARGET)
 
 relabel:
 	/sbin/restorecon $(SHLIBDIR)/$(LIBSO)
